src/data-browser/graphs/quarterly/index.jsx

```diff
@@ -37,8 +49,20 @@ export const QuarterlyGraphs = props => {
             history: props.history,
           }}
         />
```

Comment:
This can be moved into the `<Switch>` below as kind of a default route.  It would be the last `<Route>` in the `<Switch>` and it wouldn't need a `path`.  
 
 That would also allow us to remove `show={section === 'Graphs'}`. 
  
  ```
            <Route
            render={() => (
              <SectionGraphs
                {...{
                  error,
                  setError,
                  setGraphHeaderOverview,
                  location: props.location,
                  match: props.match,
                  history: props.history,
                }}
              />
            )}
          />
        </Switch>
        ```

Reply:
Sorry for sending you down that rabbit hole.  I also tried to debug this but couldn't figure out exactly which piece was doing the resetting.  I've refactored to eliminate the unmounting/mounting when switching from/to the Graph tab.